src/Components/main.css

```diff
@@ -1,3 +1,37 @@
+body{
+    font-family: Avenir;
+}
+
+.title_img{
+    background-image: url("/cryptography.jpg");
+}
+
+.title_design{
+    font-weight: bold;
+    font-size: 100px;
+    color: white;
+    padding-top: 200px;
+}
+
+.desc{
+    font-weight: normal;
+    font-size: 48px;
+    color: white;
+    padding: 50px;
+    width: 60%;
+    margin: auto;
+    position: relative;
+    text-align: center;
+    padding-bottom: 350px;
+}
```

Comment:
While lots of custom CSS isn't inherently a bad thing, it might be a good idea to review some of the bulma docs for the description to keep your landing responsive (see https://bulma.io/documentation/elements/title/'s subtitle classes), and then enforce a minimum height on the title page to make it bleed over a bit into the content more than usual. This way, you don't have to worry about the specifics of styling your subtitle, and ultimately write less code!

This same feedback can be applied at large to the custom CSS that we see here - you don't necessarily need to write a `.width50` class when bulma allows you to control the relative size of items in a section/container.